test-cypress/integration/help/rest-api.spec.js

```diff
@@ -0,0 +1,110 @@
+describe('Help / REST API', () => {
+
+    before(() => {
+        // Disables the security (only for this spec) to be able to interact with Swagger's iframe
+        // See https://github.com/cypress-io/cypress/issues/136
+        Cypress.config({chromeWebSecurity: false});
+    });
+
+    beforeEach(() => {
+        cy.visit('/webapi');
+        getSwaggerFrame()
+            .find('#message-bar')
+            .should('be.visible');
+    });
+
+    // TODO: Check explicitly for all controllers (name + summary) ?
```

Comment:
Currently I test only single controller in each section. It could easily be extended to test each one of them.